package.json

```diff
@@ -53,20 +57,21 @@
     "babel-cli": "^6.18.0",
     "babel-core": "^6.21.0",
     "babel-eslint": "^7.1.1",
-    "babel-jest": "^18.0.0",
+    "babel-jest": "^20.0.3",
     "babel-plugin-syntax-object-rest-spread": "^6.13.0",
     "babel-plugin-transform-object-rest-spread": "^6.20.2",
     "babel-preset-es2015": "^6.18.0",
     "eslint": "^3.13.1",
     "eslint-config-airbnb": "^14.0.0",
     "eslint-plugin-import": "^2.2.0",
+    "eslint-plugin-jest": "^20.0.3",
```

Comment:
kk I can't really say, bc I don't use it myself and as a StandardJS user I never really dealt with ESLint aswell :), if ESLint complains it needs t be solved either in `eslint-config-webpack` or via a PR adding this plugin to `webpack-defaults`in te long-term